src/main/java/com/rackspacecloud/metrics/queryservice/services/RequestLogging.java

```diff
@@ -16,6 +17,21 @@
 @Slf4j
 public class RequestLogging extends HandlerInterceptorAdapter {
 
+  /* Additional unsafe logging */
+  /*
+  @Override
+  public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+    log.info("request url:{}", request.getRequestURI());
+
+    Enumeration<String> headerNames = request.getHeaderNames();
+    String name;
+    while(headerNames.hasMoreElements()) {
+      name = headerNames.nextElement();
+      log.info("header: {}:{}", name, request.getHeader(name));
+    }
+    return true;
+  }*/
```

Comment:
Did you still want to keep this around? If so, might be able to use Spring profiles or a property to conditionalize.

Reply:
I considered making this conditional on the dev profile, but was still a bit worried about it. Essentially this logs security tokens when enabled.